src/main/src/main/java/org/geoserver/config/util/XStreamPersister.java

```diff
@@ -2710,6 +2718,19 @@ public Object doUnmarshal(
         }
     }
 
+    class SimpleInternationalStringConverter extends AbstractSingleValueConverter {
```

Comment:
The other international strings do not have such problem, but I've found the implemnetation is always using GrowableInternationalString for the fields, and performing conversions on the fly:

 ```
this.internationalTitle = InternationalStringUtils.growable(internationalTitle);
```

So I guess the "class" attribute is not there because the implementation class is fixed.

Reply:
switched to growable and original converter